File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    HashMap<Integer, Integer> map = new HashMap<>();
    public boolean hasDuplicate(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            if (map.containsKey(nums[i])) {
                return true;
            }
            else {
                map.put(nums[i], 1);
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hashmap = defaultdict(int)
        maxCount = majorityNum = 0
        
        for i in nums:
            hashmap[i] += 1
            if maxCount < hashmap[i]:
                maxCount = hashmap[i]
                majorityNum = i

        return majorityNum